app/Config/core.php

```diff
@@ -48,11 +48,11 @@
  *
  * @see ErrorHandler for more information on error handling and configuration.
  */
-	Configure::write('Error', array(
-		'handler' => 'ErrorHandler::handleError',
-		'level' => E_ALL & ~E_DEPRECATED,
-		'trace' => true
-	));
+    Configure::write('Error', array(
+        'handler' => 'ErrorHandler::handleError',
+        'level' => E_ALL & ~E_DEPRECATED,
+        'trace' => true
+    ));
```

Comment:
__[staging][code_sniffer]__
* Closing parenthesis not aligned correctly; expected 30 space(s) but found 4
* Closing parenthesis of a multi-line function call must be on a line by itself